### ISSUE TITLE
Level Viewer crashes on selecting any other map

### ISSUE THREAD

**Issue by Nitestokeh** [AdrianoDiDio/Medal-Of-Honor-PSX-File-Viewer#4]
Greetings.
When selecting any level but the starting one, the program hangs on "Ready", then closes itself.
![image](https://user-images.githubusercontent.com/11159125/171758445-da11304c-b25f-4846-8d08-96d2f4398958.png)

**Comment by AdrianoDiDio** [comment]
Hello, I've managed to replicate the issue and seems to be caused by a bug in the SDL library when using mouse capture on windows.
This issue seems to have been solved in the latest SDL version, so I'll try to update the library and create a new release.

**Comment by Nitestokeh** [comment]
Wonderful!
On a side note, are there plans to include player models/first person weapons in BSD showcase rendering?

**Comment by AdrianoDiDio** [comment]
I've created a new release that uses the latest SDL version.
I've tested it under Windows and it doesn't seems to crash anymore...let me know if you find any problem.
Player/Enemy/Weapons model are a planned feature but due to their complexity will likely take a fair amount of time to complete, since the format used is quite different from the one used for the other geometry ( like the level files or the objects).
